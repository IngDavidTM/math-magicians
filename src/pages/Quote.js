import React from 'react';

const Quote = () => (
  <div>
    <h2>
      Mathemathics is not about numbers, equations, computations, or algorithms;
      it is about undertanding. -William Paul Thurston
    </h2>
  </div>
);

export default Quote;
