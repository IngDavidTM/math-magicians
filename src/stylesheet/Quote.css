.quote {
  height: 80vh;
  display: flex;
  align-items: center;
  justify-content: center;
}

.quote h2 {
  max-width: 1000px;
  width: 80%;
  font-size: 26px;
  text-align: justify;
  background-color: #ffffffc5;
  backdrop-filter: blur(5px);
  border-radius: 5px;
}
