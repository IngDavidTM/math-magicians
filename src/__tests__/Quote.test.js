import renderer from "react-test-renderer";
import Quote from "../pages/Quote";

it("should render Quote", () => {
  const renderQuote = renderer.create(<Quote />).toJSON();
  expect(renderQuote).toMatchSnapshot();
});
