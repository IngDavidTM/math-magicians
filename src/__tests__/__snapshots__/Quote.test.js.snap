// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Quote 1`] = `
<div
  className="quote"
>
  <h2>
    Mathemathics is not about numbers, equations, computations, or algorithms; it is about undertanding. -William Paul Thurston
  </h2>
</div>
`;
